rpc-client-api/src/request.rs

```diff
@@ -8,9 +8,7 @@ use {
 
 #[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]
 pub enum RpcRequest {
-    Custom {
-        method: &'static str,
-    },
+    Custom { method: &'static str },
```

Comment:
rustfmt, you are weird, you [just changed this](https://github.com/anza-xyz/agave/pull/2005)

Reply:
i like when it touches lines that haven't otherwise been touched by the diff. it's good for the culture